internal/config/optionalboolean.go

```diff
@@ -20,6 +21,11 @@ type OptionalBoolean int
 // an error when trying to set a non-Unset OptionalBoolean
 
 const (
+	// FalseFlagValue false Bool String value
```

Comment:
The comment could be more descriptive about what it actually does; maybe something like 'FalseFlagValue is the command-line flag string for false values'?